Add 'Circled Information Source' symbol to Moneropedia links

erciccione opened the pull request:
![Screenshot from 2020-11-04 14-59-45](https://user-images.githubusercontent.com/28106476/98127277-6904d800-1eae-11eb-911b-6a8bb098567a.png)


Tested on Firefox and Chromium. Please test it on more browsers.

I was thinking of removing the dotted bottom to the internal Moneropedia links. Mostly because i don't like that in general and i thought the link to the Moneropedia entry would look nicer only in bold + the circled 'i' logo. I decided to keep the dotted bottom here only for consistency with the other internal links, but if people agree it would look nicer without, i'll take it out with pleasure :)

erciccione commented:
Converted to draft because i'm currently working on changing the way external links are displayed and i want to make consistent styling choices.

erciccione commented:
Showing external links automatically on GetMonero it's trickier than i anticipated, Let's merge this in the meantime, we can adapt the styling later if necessary.

erciccione commented:
Showing external links automatically on GetMonero it's trickier than i anticipated, Let's merge this in the meantime, we can adapt the styling later if necessary.

erciccione commented:
Rebased to update the preview. This still needs to be reviewed.

selsta commented:
<img width="136" alt="Screenshot 2020-10-18 at 23 14 59" src="https://user-images.githubusercontent.com/7697454/96385979-c0f5cc00-1197-11eb-93b8-4d917ecd8b3f.png">

Looks like this in Safari

erciccione commented:
@selsta could you test now?

selsta commented:
same hmm

erciccione commented:
I have no clue. Are we sure it's not a local issue? Maybe fonts or something else?
Safari should support Unicode entities.

00-matt commented:
[Also broken in Chrome on macOS](https://i.imgur.com/uDGUgsg.png). Just a guess, but maybe you could use `content: "\1F6C8"` in the CSS instead??

erciccione commented:
> maybe you could use content: "\1F6C8" in the CSS instead?

That's what i did before force pushing the last change. Using `\1F6C8` should give less compatibility problems reguardless, but i thought it worthed a try.

erciccione commented:
The icon seems to work correctly in my local machine in Firefox and chromium, but doesn't work on my ubuntu VM in chromium, chrome and firefox. I'm considering switching to just use an SVG. Making this PR a draft until i figure it out.

erciccione commented:
The problem was not related to browsers but to fonts. Open sans doesn't include that unicode character so you either have it installed locally or it won't show up. The only reliable solution is to use an svg image, so i uploaded one.

To add some stylistic diversity i removed the dotted line under these links. Now there is the usual continuous line used for links on hover. I updated the screenshot.

This is now ready to be reviewed.
